dashboard/pkg/epinio/l10n/en-us.yaml

```diff
@@ -284,6 +284,8 @@ epinio:
         label: Download Manifest
       editFromCommit:
         label: Redeploy
+      goToEpinio:
+        label: App Details
```

Comment:
Didn't spot this before. This should be labelled `Epinio App`, possibly `Epinio Details`? Either way, think it should be clearer where the user ends up

Reply:
I'm in favor of `Epinio App`. Epinio Details i think is too long.